meteo-notebook.ipynb

```diff
@@ -110,7 +110,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    plt.title('Scatter Plot')
</pre><p>Give it a proper title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>